packages/contracts/src/utils/JwtVerifier.sol

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "../interfaces/IJwtGroth16Verifier.sol";
+import {UUPSUpgradeable} from "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { strings } from "solidity-stringutils/src/strings.sol";
+
+struct EmailProof {
+    string domainName; // Domain name of the sender's email
+    bytes32 publicKeyHash; // Hash of the DKIM public key used in email/proof
+    uint timestamp; // Timestamp of the email
+    string maskedCommand; // Masked command of the email
+    bytes32 emailNullifier; // Nullifier of the email to prevent its reuse.
```

Comment:
```suggestion
    bytes32 jwtNullifier; // Nullifier of the jwt to prevent its reuse.
```

Reply:
@shreyas-londhe I've not dared to make any changes here in order to be compatible with EmailProof already in use.
If this change is made, it will not be possible to use the existing Verifier and JwtVerifier in the same interface with IVerifier used in ether-email-auth.